py2store/persisters/_arangodb_in_progress.py

```diff
@@ -0,0 +1,136 @@
+from collections.abc import Iterable
+from py2store.base import Persister
+from pyArango.connection import Connection
+
+
+class ArangoDbPersister(Persister):
+    """
+    A basic ArangoDB persister.
+    >>> from py2store.persisters._arangodb_in_progress import ArangoDbPersister
+    >>> s = ArangoDbPersister()
+    >>> k = {'key': '777'} # Each collection will happily accept user-defined _key values.
+    >>> v = {'val': 'bar'}
+    >>> for _key in s:
+    ...     del s[_key]
+    ...
+    >>> k in s
+    False
+    >>> len(s)
+    0
+    >>> s[k] = v
+    >>> len(s)
+    1
+    >>> s[k]
+    {'val': 'bar'}
+    >>> s.get(k)
+    {'val': 'bar'}
+    >>> s.get({'not': 'a key'}, {'default': 'val'})  # testing s.get with default
+    {'default': 'val'}
+    >>> list(s.values())
+    [{'val': 'bar'}]
+    >>> k in s  # testing __contains__ again
+    True
+    >>> del s[k]
+    >>> len(s)
+    0
+    >>> s = ArangoDbPersister(db_name='py2store', key_fields=('name',), data_fields=('yob', 'proj', 'bdfl'))
+    >>> for _key in s:
+    ...     del s[_key]
+    ...
+    >>> s[{'name': 'guido'}] = {'yob': 1956, 'proj': 'python', 'bdfl': False}
+    >>> s[{'name': 'guido'}]
+    {'yob': 1956, 'proj': 'python', 'bdfl': False}
+    >>> s[{'name': 'vitalik'}] = {'yob': 1994, 'proj': 'ethereum', 'bdfl': True}
+    >>> s[{'name': 'vitalik'}]
+    {'yob': 1994, 'proj': 'ethereum', 'bdfl': True}
+    >>> for key, val in s.items():
+    ...     print(f"{key}: {val}")
+    {'name': 'guido'}: {'yob': 1956, 'proj': 'python', 'bdfl': False}
+    {'name': 'vitalik'}: {'yob': 1994, 'proj': 'ethereum', 'bdfl': True}
+    """
+
+    def clear(self):
```

Comment:
This method is defined in the parent class, are you sure you need to redefine it here?